### DISCUSSION TITLE
Add ability to configure `app.use(bodyParser.json({limit: process.env.MAX_BODY_SIZE }));` via express in app.ts.

### DISCUSSION THREAD

**Discussion by ipbyrne** [directus/directus#13097] [Feature Requests]
I have an issue where I want to limit the max file size that can be uploaded. 

This can easily be done in express, but Directus doesn't seem to allow this.

Inside this file, we could simply add something like this underline 156 here: https://github.com/directus/directus/blob/main/api/src/app.ts#L156
```
if (env.MAX_BODY_SIZE) {
    app.use(bodyParser.json({limit: env.MAX_BODY_SIZE }));
}
```

**Comment by rijkvanzanten** [comment]
Your example code already exists:

https://github.com/directus/directus/blob/e47502e11fc173f7cb9524fc6e853de548dd8389/api/src/app.ts#L133-L145

but doesn't limit file uploads, as those aren't send as JSON payloads.

> This can easily be done in express

Do you have an example? The best I can come up with is to set a max limit on how many bytes to parse in busboy, throw an error when that limit is reached, and undo the upload so far. Seeing that uploads rely on Streams, this might be less trivially easy than you'd think

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
